buildModels.js

```diff
@@ -0,0 +1,29 @@
+var fs = require('fs');
```

Comment:
Err...really? In the viewer we use dashes for the fragment and vertex shader filenames, and in kokorigami, we have a dash in the gl-matrix filename. :P